### ISSUE TITLE
Feedback on mini_ex5 by Rolf

### ISSUE THREAD

**Issue by OorthianEmissary** [Emmapilk/Aestetisk-programmering-2017#8]
This program may talk about a lot of shit, but it surely is not itself.

Without having looked at your readMe, I opened the program, only to be amused by it, as I completely understood the context, where the found text was from and how this was relayed to the user, which in this case was me. Using only the p5.js library, I suppose you have used random() to shuffle through your string/array of found text. This is, again, without having looked at your code. One thing about the found text though is that after having shuffled through a handful of them, you begin seeing once you have already seen. I can see that adding maybe 100+ lines of text found in bathroomstalls would have been troublesome though. As a proof of concept, it is more than sufficient. When the program is opened, the picture instantly sets the scene, and with the paragraph above gives the atmosphere and context.
A way you could have integrated the RiTaLib might be by using the RiMarkov-class function for text analysis and making n-strings that would make new loo quotes.

In conclusion. Talks shit, but isn't.
